Build on develop or 1.0.0 fails with Intel 19 compilers

**Describe the bug**
```
[  0%] Building CXX object src/CMakeFiles/vt.dir/vt/configs/error/pretty_print_message.cc.o
/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h(1143): error #137: expression must be a modifiable lvalue
    if (next_arg_id_ >= 0) return internal::to_unsigned(next_arg_id_++);
                                                        ^
          detected during:
            instantiation of "unsigned int fmt::v5::basic_parse_context<Char, ErrorHandler>::next_arg_id() const [with Char=char, ErrorHandler=fmt::v5::internal::error_handler]" at line 3261
            instantiation of "void fmt::v5::format_handler<ArgFormatter, Char, Context>::on_arg_id() [with ArgFormatter=fmt::v5::arg_formatter<fmt::v5::back_insert_range<fmt::v5::internal::buffer<char>>>, Char=char, Context=fmt::v5::format_context]" at line 2177
            instantiation of "void fmt::v5::internal::parse_format_string<IS_CONSTEXPR,Char,Handler>(fmt::v5::basic_string_view<Char>, Handler &&) [with IS_CONSTEXPR=false, Char=char, Handler=fmt::v5::format_handler<fmt::v5::arg_formatter<fmt::v5::back_insert_range<fmt::v5::internal::buffer<char>>>, char, fmt::v5::format_context> &]" at line 3306
            instantiation of "Context::iterator fmt::v5::vformat_to<ArgFormatter,Char,Context>(ArgFormatter::range, fmt::v5::basic_string_view<Char>, fmt::v5::basic_format_args<Context>, fmt::v5::internal::locale_ref) [with ArgFormatter=fmt::v5::arg_formatter<fmt::v5::back_insert_range<fmt::v5::internal::buffer<char>>>, Char=char, Context=fmt::v5::format_context]" at line 3404
            instantiation of "fmt::v5::buffer_context<Char>::type::iterator fmt::v5::internal::vformat_to(fmt::v5::internal::buffer<Char> &, fmt::v5::basic_string_view<Char>, fmt::v5::basic_format_args<fmt::v5::buffer_context<Char>::type>) [with Char=char]" at line 928 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format-inl.h"

/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/core.h(572): error: the object has cv-qualifiers that are not compatible with the member function "fmt::v5::internal::error_handler::on_error"
            object type is: const fmt::v5::internal::error_handler
      ErrorHandler::on_error(message);
      ^
          detected during:
            instantiation of "void fmt::v5::basic_parse_context<Char, ErrorHandler>::on_error(const char *) const [with Char=char, ErrorHandler=fmt::v5::internal::error_handler]" at line 1144 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h"
            instantiation of "unsigned int fmt::v5::basic_parse_context<Char, ErrorHandler>::next_arg_id() const [with Char=char, ErrorHandler=fmt::v5::internal::error_handler]" at line 3261 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h"
            instantiation of "void fmt::v5::format_handler<ArgFormatter, Char, Context>::on_arg_id() [with ArgFormatter=fmt::v5::arg_formatter<fmt::v5::back_insert_range<fmt::v5::internal::buffer<char>>>, Char=char, Context=fmt::v5::format_context]" at line 2177 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h"
            instantiation of "void fmt::v5::internal::parse_format_string<IS_CONSTEXPR,Char,Handler>(fmt::v5::basic_string_view<Char>, Handler &&) [with IS_CONSTEXPR=false, Char=char, Handler=fmt::v5::format_handler<fmt::v5::arg_formatter<fmt::v5::back_insert_range<fmt::v5::internal::buffer<char>>>, char, fmt::v5::format_context> &]" at line 3306 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h"
            instantiation of "Context::iterator fmt::v5::vformat_to<ArgFormatter,Char,Context>(ArgFormatter::range, fmt::v5::basic_string_view<Char>, fmt::v5::basic_format_args<Context>, fmt::v5::internal::locale_ref) [with ArgFormatter=fmt::v5::arg_formatter<fmt::v5::back_insert_range<fmt::v5::internal::buffer<char>>>, Char=char, Context=fmt::v5::format_context]" at line 3404 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h"
            instantiation of "fmt::v5::buffer_context<Char>::type::iterator fmt::v5::internal::vformat_to(fmt::v5::internal::buffer<Char> &, fmt::v5::basic_string_view<Char>, fmt::v5::basic_format_args<fmt::v5::buffer_context<Char>::type>) [with Char=char]" at line 928 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format-inl.h"

/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/core.h(433): error #137: expression must be a modifiable lvalue
      data_ += n;
      ^
          detected during:
            instantiation of "void fmt::v5::basic_string_view<Char>::remove_prefix(size_t={unsigned long}) const [with Char=char]" at line 554
            instantiation of "void fmt::v5::basic_parse_context<Char, ErrorHandler>::advance_to(fmt::v5::basic_parse_context<Char, ErrorHandler>::iterator) const [with Char=char, ErrorHandler=fmt::v5::internal::error_handler]" at line 3269 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h"
            instantiation of "void fmt::v5::format_handler<ArgFormatter, Char, Context>::on_replacement_field(const Char *) [with ArgFormatter=fmt::v5::arg_formatter<fmt::v5::back_insert_range<fmt::v5::internal::buffer<char>>>, Char=char, Context=fmt::v5::format_context]" at line 2178 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h"
            instantiation of "void fmt::v5::internal::parse_format_string<IS_CONSTEXPR,Char,Handler>(fmt::v5::basic_string_view<Char>, Handler &&) [with IS_CONSTEXPR=false, Char=char, Handler=fmt::v5::format_handler<fmt::v5::arg_formatter<fmt::v5::back_insert_range<fmt::v5::internal::buffer<char>>>, char, fmt::v5::format_context> &]" at line 3306 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h"
            instantiation of "Context::iterator fmt::v5::vformat_to<ArgFormatter,Char,Context>(ArgFormatter::range, fmt::v5::basic_string_view<Char>, fmt::v5::basic_format_args<Context>, fmt::v5::internal::locale_ref) [with ArgFormatter=fmt::v5::arg_formatter<fmt::v5::back_insert_range<fmt::v5::internal::buffer<char>>>, Char=char, Context=fmt::v5::format_context]" at line 3404 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h"
            instantiation of "fmt::v5::buffer_context<Char>::type::iterator fmt::v5::internal::vformat_to(fmt::v5::internal::buffer<Char> &, fmt::v5::basic_string_view<Char>, fmt::v5::basic_format_args<fmt::v5::buffer_context<Char>::type>) [with Char=char]" at line 928 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format-inl.h"

/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/core.h(434): error #137: expression must be a modifiable lvalue
      size_ -= n;
      ^
          detected during:
            instantiation of "void fmt::v5::basic_string_view<Char>::remove_prefix(size_t={unsigned long}) const [with Char=char]" at line 554
            instantiation of "void fmt::v5::basic_parse_context<Char, ErrorHandler>::advance_to(fmt::v5::basic_parse_context<Char, ErrorHandler>::iterator) const [with Char=char, ErrorHandler=fmt::v5::internal::error_handler]" at line 3269 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h"
            instantiation of "void fmt::v5::format_handler<ArgFormatter, Char, Context>::on_replacement_field(const Char *) [with ArgFormatter=fmt::v5::arg_formatter<fmt::v5::back_insert_range<fmt::v5::internal::buffer<char>>>, Char=char, Context=fmt::v5::format_context]" at line 2178 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h"
            instantiation of "void fmt::v5::internal::parse_format_string<IS_CONSTEXPR,Char,Handler>(fmt::v5::basic_string_view<Char>, Handler &&) [with IS_CONSTEXPR=false, Char=char, Handler=fmt::v5::format_handler<fmt::v5::arg_formatter<fmt::v5::back_insert_range<fmt::v5::internal::buffer<char>>>, char, fmt::v5::format_context> &]" at line 3306 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h"
            instantiation of "Context::iterator fmt::v5::vformat_to<ArgFormatter,Char,Context>(ArgFormatter::range, fmt::v5::basic_string_view<Char>, fmt::v5::basic_format_args<Context>, fmt::v5::internal::locale_ref) [with ArgFormatter=fmt::v5::arg_formatter<fmt::v5::back_insert_range<fmt::v5::internal::buffer<char>>>, Char=char, Context=fmt::v5::format_context]" at line 3404 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h"
            instantiation of "fmt::v5::buffer_context<Char>::type::iterator fmt::v5::internal::vformat_to(fmt::v5::internal::buffer<Char> &, fmt::v5::basic_string_view<Char>, fmt::v5::basic_format_args<fmt::v5::buffer_context<Char>::type>) [with Char=char]" at line 928 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format-inl.h"

/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/core.h(565): error #137: expression must be a modifiable lvalue
      next_arg_id_ = -1;
      ^
          detected during:
            instantiation of "bool fmt::v5::basic_parse_context<Char, ErrorHandler>::check_arg_id(unsigned int) const [with Char=char, ErrorHandler=fmt::v5::internal::error_handler]" at line 3263 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h"
            instantiation of "void fmt::v5::format_handler<ArgFormatter, Char, Context>::on_arg_id(unsigned int) [with ArgFormatter=fmt::v5::arg_formatter<fmt::v5::back_insert_range<fmt::v5::internal::buffer<char>>>, Char=char, Context=fmt::v5::format_context]" at line 2136 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h"
            instantiation of "void fmt::v5::internal::id_adapter<Handler, Char>::operator()(unsigned int) const [with Handler=fmt::v5::format_handler<fmt::v5::arg_formatter<fmt::v5::back_insert_range<fmt::v5::internal::buffer<char>>>, char, fmt::v5::format_context> &, Char=char]" at line 1939 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h"
            instantiation of "const Char *fmt::v5::internal::parse_arg_id(const Char *, const Char *, IDHandler &&) [with Char=char, IDHandler=fmt::v5::internal::id_adapter<fmt::v5::format_handler<fmt::v5::arg_formatter<fmt::v5::back_insert_range<fmt::v5::internal::buffer<char>>>, char, fmt::v5::format_context> &, char>]" at line 2182 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h"
            instantiation of "void fmt::v5::internal::parse_format_string<IS_CONSTEXPR,Char,Handler>(fmt::v5::basic_string_view<Char>, Handler &&) [with IS_CONSTEXPR=false, Char=char, Handler=fmt::v5::format_handler<fmt::v5::arg_formatter<fmt::v5::back_insert_range<fmt::v5::internal::buffer<char>>>, char, fmt::v5::format_context> &]" at line 3306 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h"
            instantiation of "Context::iterator fmt::v5::vformat_to<ArgFormatter,Char,Context>(ArgFormatter::range, fmt::v5::basic_string_view<Char>, fmt::v5::basic_format_args<Context>, fmt::v5::internal::locale_ref) [with ArgFormatter=fmt::v5::arg_formatter<fmt::v5::back_insert_range<fmt::v5::internal::buffer<char>>>, Char=char, Context=fmt::v5::format_context]" at line 3404 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h"
            instantiation of "fmt::v5::buffer_context<Char>::type::iterator fmt::v5::internal::vformat_to(fmt::v5::internal::buffer<Char> &, fmt::v5::basic_string_view<Char>, fmt::v5::basic_format_args<fmt::v5::buffer_context<Char>::type>) [with Char=char]" at line 928 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format-inl.h"

/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h(1143): error #137: expression must be a modifiable lvalue
    if (next_arg_id_ >= 0) return internal::to_unsigned(next_arg_id_++);
                                                        ^
          detected during:
            instantiation of "unsigned int fmt::v5::basic_parse_context<Char, ErrorHandler>::next_arg_id() const [with Char=wchar_t, ErrorHandler=fmt::v5::internal::error_handler]" at line 3261
            instantiation of "void fmt::v5::format_handler<ArgFormatter, Char, Context>::on_arg_id() [with ArgFormatter=fmt::v5::arg_formatter<fmt::v5::back_insert_range<fmt::v5::internal::buffer<wchar_t>>>, Char=wchar_t, Context=fmt::v5::wformat_context]" at line 2177
            instantiation of "void fmt::v5::internal::parse_format_string<IS_CONSTEXPR,Char,Handler>(fmt::v5::basic_string_view<Char>, Handler &&) [with IS_CONSTEXPR=false, Char=wchar_t, Handler=fmt::v5::format_handler<fmt::v5::arg_formatter<fmt::v5::back_insert_range<fmt::v5::internal::buffer<wchar_t>>>, wchar_t, fmt::v5::wformat_context> &]" at line 3306
            instantiation of "Context::iterator fmt::v5::vformat_to<ArgFormatter,Char,Context>(ArgFormatter::range, fmt::v5::basic_string_view<Char>, fmt::v5::basic_format_args<Context>, fmt::v5::internal::locale_ref) [with ArgFormatter=fmt::v5::arg_formatter<fmt::v5::back_insert_range<fmt::v5::internal::buffer<wchar_t>>>, Char=wchar_t, Context=fmt::v5::wformat_context]" at line 3404
            instantiation of "fmt::v5::buffer_context<Char>::type::iterator fmt::v5::internal::vformat_to(fmt::v5::internal::buffer<Char> &, fmt::v5::basic_string_view<Char>, fmt::v5::basic_format_args<fmt::v5::buffer_context<Char>::type>) [with Char=wchar_t]" at line 935 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format-inl.h"

/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/core.h(572): error: the object has cv-qualifiers that are not compatible with the member function "fmt::v5::internal::error_handler::on_error"
            object type is: const fmt::v5::internal::error_handler
      ErrorHandler::on_error(message);
      ^
          detected during:
            instantiation of "void fmt::v5::basic_parse_context<Char, ErrorHandler>::on_error(const char *) const [with Char=wchar_t, ErrorHandler=fmt::v5::internal::error_handler]" at line 1144 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h"
            instantiation of "unsigned int fmt::v5::basic_parse_context<Char, ErrorHandler>::next_arg_id() const [with Char=wchar_t, ErrorHandler=fmt::v5::internal::error_handler]" at line 3261 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h"
            instantiation of "void fmt::v5::format_handler<ArgFormatter, Char, Context>::on_arg_id() [with ArgFormatter=fmt::v5::arg_formatter<fmt::v5::back_insert_range<fmt::v5::internal::buffer<wchar_t>>>, Char=wchar_t, Context=fmt::v5::wformat_context]" at line 2177 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h"
            instantiation of "void fmt::v5::internal::parse_format_string<IS_CONSTEXPR,Char,Handler>(fmt::v5::basic_string_view<Char>, Handler &&) [with IS_CONSTEXPR=false, Char=wchar_t, Handler=fmt::v5::format_handler<fmt::v5::arg_formatter<fmt::v5::back_insert_range<fmt::v5::internal::buffer<wchar_t>>>, wchar_t, fmt::v5::wformat_context> &]" at line 3306 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h"
            instantiation of "Context::iterator fmt::v5::vformat_to<ArgFormatter,Char,Context>(ArgFormatter::range, fmt::v5::basic_string_view<Char>, fmt::v5::basic_format_args<Context>, fmt::v5::internal::locale_ref) [with ArgFormatter=fmt::v5::arg_formatter<fmt::v5::back_insert_range<fmt::v5::internal::buffer<wchar_t>>>, Char=wchar_t, Context=fmt::v5::wformat_context]" at line 3404 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h"
            instantiation of "fmt::v5::buffer_context<Char>::type::iterator fmt::v5::internal::vformat_to(fmt::v5::internal::buffer<Char> &, fmt::v5::basic_string_view<Char>, fmt::v5::basic_format_args<fmt::v5::buffer_context<Char>::type>) [with Char=wchar_t]" at line 935 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format-inl.h"

/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/core.h(433): error #137: expression must be a modifiable lvalue
      data_ += n;
      ^
          detected during:
            instantiation of "void fmt::v5::basic_string_view<Char>::remove_prefix(size_t={unsigned long}) const [with Char=wchar_t]" at line 554
            instantiation of "void fmt::v5::basic_parse_context<Char, ErrorHandler>::advance_to(fmt::v5::basic_parse_context<Char, ErrorHandler>::iterator) const [with Char=wchar_t, ErrorHandler=fmt::v5::internal::error_handler]" at line 3269 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h"
            instantiation of "void fmt::v5::format_handler<ArgFormatter, Char, Context>::on_replacement_field(const Char *) [with ArgFormatter=fmt::v5::arg_formatter<fmt::v5::back_insert_range<fmt::v5::internal::buffer<wchar_t>>>, Char=wchar_t, Context=fmt::v5::wformat_context]" at line 2178 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h"
            instantiation of "void fmt::v5::internal::parse_format_string<IS_CONSTEXPR,Char,Handler>(fmt::v5::basic_string_view<Char>, Handler &&) [with IS_CONSTEXPR=false, Char=wchar_t, Handler=fmt::v5::format_handler<fmt::v5::arg_formatter<fmt::v5::back_insert_range<fmt::v5::internal::buffer<wchar_t>>>, wchar_t, fmt::v5::wformat_context> &]" at line 3306 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h"
            instantiation of "Context::iterator fmt::v5::vformat_to<ArgFormatter,Char,Context>(ArgFormatter::range, fmt::v5::basic_string_view<Char>, fmt::v5::basic_format_args<Context>, fmt::v5::internal::locale_ref) [with ArgFormatter=fmt::v5::arg_formatter<fmt::v5::back_insert_range<fmt::v5::internal::buffer<wchar_t>>>, Char=wchar_t, Context=fmt::v5::wformat_context]" at line 3404 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h"
            instantiation of "fmt::v5::buffer_context<Char>::type::iterator fmt::v5::internal::vformat_to(fmt::v5::internal::buffer<Char> &, fmt::v5::basic_string_view<Char>, fmt::v5::basic_format_args<fmt::v5::buffer_context<Char>::type>) [with Char=wchar_t]" at line 935 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format-inl.h"

/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/core.h(434): error #137: expression must be a modifiable lvalue
      size_ -= n;
      ^
          detected during:
            instantiation of "void fmt::v5::basic_string_view<Char>::remove_prefix(size_t={unsigned long}) const [with Char=wchar_t]" at line 554
            instantiation of "void fmt::v5::basic_parse_context<Char, ErrorHandler>::advance_to(fmt::v5::basic_parse_context<Char, ErrorHandler>::iterator) const [with Char=wchar_t, ErrorHandler=fmt::v5::internal::error_handler]" at line 3269 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h"
            instantiation of "void fmt::v5::format_handler<ArgFormatter, Char, Context>::on_replacement_field(const Char *) [with ArgFormatter=fmt::v5::arg_formatter<fmt::v5::back_insert_range<fmt::v5::internal::buffer<wchar_t>>>, Char=wchar_t, Context=fmt::v5::wformat_context]" at line 2178 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h"
            instantiation of "void fmt::v5::internal::parse_format_string<IS_CONSTEXPR,Char,Handler>(fmt::v5::basic_string_view<Char>, Handler &&) [with IS_CONSTEXPR=false, Char=wchar_t, Handler=fmt::v5::format_handler<fmt::v5::arg_formatter<fmt::v5::back_insert_range<fmt::v5::internal::buffer<wchar_t>>>, wchar_t, fmt::v5::wformat_context> &]" at line 3306 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h"
            instantiation of "Context::iterator fmt::v5::vformat_to<ArgFormatter,Char,Context>(ArgFormatter::range, fmt::v5::basic_string_view<Char>, fmt::v5::basic_format_args<Context>, fmt::v5::internal::locale_ref) [with ArgFormatter=fmt::v5::arg_formatter<fmt::v5::back_insert_range<fmt::v5::internal::buffer<wchar_t>>>, Char=wchar_t, Context=fmt::v5::wformat_context]" at line 3404 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h"
            instantiation of "fmt::v5::buffer_context<Char>::type::iterator fmt::v5::internal::vformat_to(fmt::v5::internal::buffer<Char> &, fmt::v5::basic_string_view<Char>, fmt::v5::basic_format_args<fmt::v5::buffer_context<Char>::type>) [with Char=wchar_t]" at line 935 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format-inl.h"

/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/core.h(565): error #137: expression must be a modifiable lvalue
      next_arg_id_ = -1;
      ^
          detected during:
            instantiation of "bool fmt::v5::basic_parse_context<Char, ErrorHandler>::check_arg_id(unsigned int) const [with Char=wchar_t, ErrorHandler=fmt::v5::internal::error_handler]" at line 3263 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h"
            instantiation of "void fmt::v5::format_handler<ArgFormatter, Char, Context>::on_arg_id(unsigned int) [with ArgFormatter=fmt::v5::arg_formatter<fmt::v5::back_insert_range<fmt::v5::internal::buffer<wchar_t>>>, Char=wchar_t, Context=fmt::v5::wformat_context]" at line 2136 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h"
            instantiation of "void fmt::v5::internal::id_adapter<Handler, Char>::operator()(unsigned int) const [with Handler=fmt::v5::format_handler<fmt::v5::arg_formatter<fmt::v5::back_insert_range<fmt::v5::internal::buffer<wchar_t>>>, wchar_t, fmt::v5::wformat_context> &, Char=wchar_t]" at line 1939 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h"
            instantiation of "const Char *fmt::v5::internal::parse_arg_id(const Char *, const Char *, IDHandler &&) [with Char=wchar_t, IDHandler=fmt::v5::internal::id_adapter<fmt::v5::format_handler<fmt::v5::arg_formatter<fmt::v5::back_insert_range<fmt::v5::internal::buffer<wchar_t>>>, wchar_t, fmt::v5::wformat_context> &, wchar_t>]" at line 2182 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h"
            instantiation of "void fmt::v5::internal::parse_format_string<IS_CONSTEXPR,Char,Handler>(fmt::v5::basic_string_view<Char>, Handler &&) [with IS_CONSTEXPR=false, Char=wchar_t, Handler=fmt::v5::format_handler<fmt::v5::arg_formatter<fmt::v5::back_insert_range<fmt::v5::internal::buffer<wchar_t>>>, wchar_t, fmt::v5::wformat_context> &]" at line 3306 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h"
            instantiation of "Context::iterator fmt::v5::vformat_to<ArgFormatter,Char,Context>(ArgFormatter::range, fmt::v5::basic_string_view<Char>, fmt::v5::basic_format_args<Context>, fmt::v5::internal::locale_ref) [with ArgFormatter=fmt::v5::arg_formatter<fmt::v5::back_insert_range<fmt::v5::internal::buffer<wchar_t>>>, Char=wchar_t, Context=fmt::v5::wformat_context]" at line 3404 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format.h"
            instantiation of "fmt::v5::buffer_context<Char>::type::iterator fmt::v5::internal::vformat_to(fmt::v5::internal::buffer<Char> &, fmt::v5::basic_string_view<Char>, fmt::v5::basic_format_args<fmt::v5::buffer_context<Char>::type>) [with Char=wchar_t]" at line 935 of "/lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/lib/fmt/fmt/format-inl.h"

compilation aborted for /lscratch1/jliffla/vt-clean/vt-auto-build/vt-release-test-message/vt/vt/src/vt/configs/error/pretty_print_message.cc (code 2)
src/CMakeFiles/vt.dir/build.make:75: recipe for target 'src/CMakeFiles/vt.dir/vt/configs/error/pretty_print_message.cc.o' failed
make[2]: *** [src/CMakeFiles/vt.dir/vt/configs/error/pretty_print_message.cc.o] Error 2
CMakeFiles/Makefile2:1221: recipe for target 'src/CMakeFiles/vt.dir/all' failed
make[1]: *** [src/CMakeFiles/vt.dir/all] Error 2
Makefile:140: recipe for target 'all' failed
make: *** [all] Error 2
```

--- a/ci/azure/azure-intel-19-ubuntu-mpich.yml
+++ b/ci/azure/azure-intel-19-ubuntu-mpich.yml
@@ -1,0 +1,145 @@
+###############################################################################
+############## Warning this is a generated file---do not modify ###############
+###############################################################################
+
+name: PR tests (intel 19, ubuntu, mpich)
+
+trigger:
+  branches:
+    include:
+      - develop
+      - 1.0.0*
+
+pr:
+  branches:
+   include:
+     - '*'
+
+
+resources:
+- repo: self
+
+variables:
+  tag: '$(Build.BuildId)'
+  REPO: lifflander1/vt
+  ARCH: amd64
+  UBUNTU: 18.04
+  COMPILER_TYPE: intel
+  COMPILER: icc-19
+  BUILD_TYPE: release
+  ULIMIT_CORE: 0
+  CODE_COVERAGE: 0
+  VT_LB: 1
+  VT_TRACE: 1
+  VT_TRACE_RT: 0
+  VT_MIMALLOC: 0
+  VT_DOCS: 0
+  VT_ASAN: 0
+  VT_POOL: 0
+  VT_EXTENDED_TESTS: 0
+  VT_UNITY_BUILD: 1
+  VT_ZOLTAN: 0
+  VT_CI_BUILD: 1
+  VT_DIAGNOSTICS: 1
+  CACHE: "$(Agent.TempDirectory)/cache/"
+  cache_name: ubuntu-intel-19-cache
+  build_root: "$(CACHE)/$(ARCH)-ubuntu-$(UBUNTU)-$(COMPILER)-cache/"
+  TS: 0
+  TS_YEAR: 0
+  TS_MONTH: 0
+  TS_DAY: 0
+
+stages:
+- stage: Build
+  displayName: Build image
+  jobs:
+  - job: Build
+    displayName: Build
+    pool:
+      vmImage: 'ubuntu-18.04'
+    timeoutInMinutes: 180
+    steps:
+      - task: Bash@3
+        displayName: Job setup
+        inputs:
+          targetType: 'inline'
+          script: |
+            echo setup
+      - task: Bash@3
+        displayName: Build timestep for caching
+        inputs:
+          targetType: 'inline'
+          script: |
+            echo 'string(TIMESTAMP current_date "%H;%M;%S" UTC)' > script
+            echo 'execute_process(COMMAND ${CMAKE_COMMAND} -E echo "${current_date}")' >> script
+            val=$(cmake -P script)
+            echo "##vso[task.setvariable variable=TS]$val"
+            echo 'string(TIMESTAMP current_date "%Y" UTC)' > script
+            echo 'execute_process(COMMAND ${CMAKE_COMMAND} -E echo "${current_date}")' >> script
+            val=$(cmake -P script)
+            echo "##vso[task.setvariable variable=TS_YEAR]$val"
+            echo 'string(TIMESTAMP current_date "%m" UTC)' > script
+            echo 'execute_process(COMMAND ${CMAKE_COMMAND} -E echo "${current_date}")' >> script
+            val=$(cmake -P script)
+            echo "##vso[task.setvariable variable=TS_MONTH]$val"
+            echo 'string(TIMESTAMP current_date "%d" UTC)' > script
+            echo 'execute_process(COMMAND ${CMAKE_COMMAND} -E echo "${current_date}")' >> script
+            val=$(cmake -P script)
+            echo "##vso[task.setvariable variable=TS_DAY]$val"
+      - task: Bash@3
+        displayName: Output timestamp for caching
+        inputs:
+          targetType: 'inline'
+          script: 'echo "my pipeline variable is $(TS) $(TS_YEAR) $(TS_MONTH) $(TS_DAY)"'
+      - task: Cache@2
+        displayName: Update cache
+        inputs:
+          securityNamespace: cache
+          key: $(Agent.OS) | "$(cache_name)" | $(TS_YEAR) | $(TS_MONTH) | $(TS_DAY) | $(TS)
+          path: '$(build_root)/ccache'
+          restoreKeys: |
+            $(Agent.OS) | "$(cache_name)" | $(TS_YEAR) | $(TS_MONTH) | $(TS_DAY)
+            $(Agent.OS) | "$(cache_name)" | $(TS_YEAR) | $(TS_MONTH)
+            $(Agent.OS) | "$(cache_name)" | $(TS_YEAR)
+            $(Agent.OS) | "$(cache_name)"
+      - task: DockerCompose@0
+        displayName: Pull container
+        inputs:
+          containerregistrytype: 'Container Registry'
+          dockerComposeFile: '**/docker-compose.yml'
+          action: 'Run a Docker Compose command'
+          dockerComposeCommand: 'pull --ignore-pull-failures ubuntu-cpp-clean'
+      - task: DockerCompose@0
+        displayName: Build container
+        inputs:
+          containerregistrytype: 'Container Registry'
+          dockerComposeFile: '**/docker-compose.yml'
+          action: 'Run a Docker Compose command'
+          dockerComposeCommand: 'run ubuntu-cpp-clean'
+      - task: DockerCompose@0
+        displayName: Push container to registry
+        inputs:
+          containerregistrytype: 'Container Registry'
+          dockerRegistryEndpoint: 'dockerRegistryConnection1'
+          dockerComposeFile: '**/docker-compose.yml'
+          action: 'Run a Docker Compose command'
+          dockerComposeCommand: 'push ubuntu-cpp-clean'
+      - task: Bash@3
+        displayName: Create artifacts
+        inputs:
+          targetType: 'inline'
+          script: |
+            zip -j $(Agent.TempDirectory)/LastTest.log.gz $(build_root)/vt/Testing/Temporary/LastTest.log
+            zip -j $(Agent.TempDirectory)/cmake-output.log.gz $(build_root)/vt/cmake-output.log
+      - task: PublishPipelineArtifact@1
+        displayName: Upload CMake test output artifact
+        inputs:
+          targetPath: '$(Agent.TempDirectory)/LastTest.log.gz'
+          artifact: 'CMakeLastTestLog'
+          publishLocation: 'pipeline'
+      - task: PublishPipelineArtifact@1
+        displayName: Upload CMake full output artifact
+        inputs:
+          targetPath: '$(Agent.TempDirectory)/cmake-output.log.gz'
+          artifact: 'CMakeOutputLog'
+          publishLocation: 'pipeline'

--- a/scripts/workflows-azure.ini
+++ b/scripts/workflows-azure.ini
@@ -42,6 +42,16 @@ vt_trace = 1
 vt_pool = 0
 vt_extended_tests = 0
 
+[PR-tests-intel-19]
+test_configuration = "intel 19, ubuntu, mpich"
+compiler_type = intel
+compiler = icc-19
+cache_name = ubuntu-intel-19-cache
+output_name = ci/azure/azure-intel-19-ubuntu-mpich.yml
+vt_trace = 1
+vt_pool = 0
+vt_extended_tests = 0
+
 [PR-tests-intel-18-03-extended]
 test_type = "PR tests extended"
 test_configuration = "intel 18.03, ubuntu, mpich"

--- a/src/vt/collective/reduce/reduce.h
+++ b/src/vt/collective/reduce/reduce.h
@@ -179,15 +179,33 @@ struct Reduce : virtual collective::tree::Tree {
   template <
     typename OpT,
     typename MsgT,
-    ActiveTypedFnType<MsgT> *f = MsgT::template msgHandler<
-      MsgT, OpT, collective::reduce::operators::ReduceCallback<MsgT>
-    >
+    ActiveTypedFnType<MsgT> *f
   >
   PendingSendType reduce(
     NodeType const& root, MsgT* msg, Callback<MsgT> cb,
     detail::ReduceStamp id = detail::ReduceStamp{},
     ReduceNumType const& num_contrib = 1
   );
+  template <
+    typename OpT,
+    typename MsgT
+  >
+  PendingSendType reduce(
+    NodeType const& root, MsgT* msg, Callback<MsgT> cb,
+    detail::ReduceStamp id = detail::ReduceStamp{},
+    ReduceNumType const& num_contrib = 1
+  )
+    {
+      return reduce<
+        OpT,
+        MsgT,
+        MsgT::template msgHandler<
+          MsgT,
+          OpT,
+          collective::reduce::operators::ReduceCallback<MsgT>
+          >
+        >(root, msg, cb, id, num_contrib);
+    }
 
   /**
    * \brief Reduce a message up the tree
@@ -203,15 +221,33 @@ struct Reduce : virtual collective::tree::Tree {
   template <
     typename OpT,
     typename MsgT,
-    ActiveTypedFnType<MsgT> *f = MsgT::template msgHandler<
-      MsgT, OpT, collective::reduce::operators::ReduceCallback<MsgT>
-    >
+    ActiveTypedFnType<MsgT> *f
   >
   detail::ReduceStamp reduceImmediate(
     NodeType const& root, MsgT* msg, Callback<MsgT> cb,
     detail::ReduceStamp id = detail::ReduceStamp{},
     ReduceNumType const& num_contrib = 1
   );
+  template <
+    typename OpT,
+    typename MsgT
+  >
+  detail::ReduceStamp reduceImmediate(
+    NodeType const& root, MsgT* msg, Callback<MsgT> cb,
+    detail::ReduceStamp id = detail::ReduceStamp{},
+    ReduceNumType const& num_contrib = 1
+  )
+  {
+    return reduceImmediate<
+      OpT,
+      MsgT,
+      MsgT::template msgHandler<
+        MsgT,
+        OpT,
+        collective::reduce::operators::ReduceCallback<MsgT>
+        >
+      >(root, msg, cb, id, num_contrib);
+  }
 
   /**
    * \brief Reduce a message up the tree with a target function on the root node
@@ -227,13 +263,31 @@ struct Reduce : virtual collective::tree::Tree {
     typename OpT,
     typename FunctorT,
     typename MsgT,
-    ActiveTypedFnType<MsgT> *f = MsgT::template msgHandler<MsgT, OpT, FunctorT>
+    ActiveTypedFnType<MsgT> *f
   >
   PendingSendType reduce(
     NodeType const& root, MsgT* msg,
     detail::ReduceStamp id = detail::ReduceStamp{},
     ReduceNumType const& num_contrib = 1
   );
+  template <
+    typename OpT,
+    typename FunctorT,
+    typename MsgT
+  >
+  PendingSendType reduce(
+    NodeType const& root, MsgT* msg,
+    detail::ReduceStamp id = detail::ReduceStamp{},
+    ReduceNumType const& num_contrib = 1
+  )
+  {
+    return reduce<
+      OpT,
+      FunctorT,
+      MsgT,
+      MsgT::template msgHandler<MsgT, OpT, FunctorT>
+      >(root, msg, id, num_contrib);
+  }
 
   /**
    * \brief Reduce a message up the tree with a target function on the root node
@@ -249,13 +303,31 @@ struct Reduce : virtual collective::tree::Tree {
     typename OpT,
     typename FunctorT,
     typename MsgT,
-    ActiveTypedFnType<MsgT> *f = MsgT::template msgHandler<MsgT, OpT, FunctorT>
+    ActiveTypedFnType<MsgT> *f
   >
   detail::ReduceStamp reduceImmediate(
     NodeType const& root, MsgT* msg,
     detail::ReduceStamp id = detail::ReduceStamp{},
     ReduceNumType const& num_contrib = 1
   );
+  template <
+    typename OpT,
+    typename FunctorT,
+    typename MsgT
+    >
+  detail::ReduceStamp reduceImmediate(
+    NodeType const& root, MsgT* msg,
+    detail::ReduceStamp id = detail::ReduceStamp{},
+    ReduceNumType const& num_contrib = 1
+  )
+  {
+    return reduceImmediate<
+      OpT,
+      FunctorT,
+      MsgT,
+      MsgT::template msgHandler<MsgT, OpT, FunctorT>
+      >(root, msg, id, num_contrib);
+  }
 
   /**
    * \internal \brief Combine in a new message for a given reduction

--- a/src/vt/collective/reduce/reduce.h
+++ b/src/vt/collective/reduce/reduce.h
@@ -199,7 +199,7 @@ struct Reduce : virtual collective::tree::Tree {
       return reduce<
         OpT,
         MsgT,
-        MsgT::template msgHandler<
+        &MsgT::template msgHandler<
           MsgT,
           OpT,
           collective::reduce::operators::ReduceCallback<MsgT>
@@ -241,7 +241,7 @@ struct Reduce : virtual collective::tree::Tree {
     return reduceImmediate<
       OpT,
       MsgT,
-      MsgT::template msgHandler<
+      &MsgT::template msgHandler<
         MsgT,
         OpT,
         collective::reduce::operators::ReduceCallback<MsgT>
@@ -285,7 +285,7 @@ struct Reduce : virtual collective::tree::Tree {
       OpT,
       FunctorT,
       MsgT,
-      MsgT::template msgHandler<MsgT, OpT, FunctorT>
+      &MsgT::template msgHandler<MsgT, OpT, FunctorT>
       >(root, msg, id, num_contrib);
   }
 
@@ -325,7 +325,7 @@ struct Reduce : virtual collective::tree::Tree {
       OpT,
       FunctorT,
       MsgT,
-      MsgT::template msgHandler<MsgT, OpT, FunctorT>
+      &MsgT::template msgHandler<MsgT, OpT, FunctorT>
       >(root, msg, id, num_contrib);
   }
 

--- a/src/vt/objgroup/proxy/proxy_objgroup.h
+++ b/src/vt/objgroup/proxy/proxy_objgroup.h
@@ -149,14 +149,30 @@ public:
   template <
     typename OpT = collective::None,
     typename MsgPtrT,
-    typename MsgT = typename util::MsgPtrType<MsgPtrT>::MsgType,
-    ActiveTypedFnType<MsgT> *f = MsgT::template msgHandler<
-      MsgT, OpT, collective::reduce::operators::ReduceCallback<MsgT>
-    >
+    typename MsgT,
+    ActiveTypedFnType<MsgT> *f
   >
   PendingSendType reduce(
     MsgPtrT msg, Callback<MsgT> cb, ReduceStamp stamp = ReduceStamp{}
   ) const;
+  template <
+    typename OpT = collective::None,
+    typename MsgPtrT,
+    typename MsgT = typename util::MsgPtrType<MsgPtrT>::MsgType
+  >
+  PendingSendType reduce(
+    MsgPtrT msg, Callback<MsgT> cb, ReduceStamp stamp = ReduceStamp{}
+  ) const
+  {
+    return reduce<
+      OpT,
+      MsgPtrT,
+      MsgT,
+      MsgT::template msgHandler<
+        MsgT, OpT, collective::reduce::operators::ReduceCallback<MsgT>
+        >
+      >(msg, cb, stamp);
+  }
 
   /**
    * \brief Reduce over the objgroup instances on each node with a functor
@@ -172,9 +188,25 @@ public:
     typename FunctorT,
     typename MsgPtrT,
     typename MsgT = typename util::MsgPtrType<MsgPtrT>::MsgType,
-    ActiveTypedFnType<MsgT> *f = MsgT::template msgHandler<MsgT, OpT, FunctorT>
+    ActiveTypedFnType<MsgT> *f
   >
   PendingSendType reduce(MsgPtrT msg, ReduceStamp stamp = ReduceStamp{}) const;
+  template <
+    typename OpT = collective::None,
+    typename FunctorT,
+    typename MsgPtrT,
+    typename MsgT = typename util::MsgPtrType<MsgPtrT>::MsgType
+  >
+  PendingSendType reduce(MsgPtrT msg, ReduceStamp stamp = ReduceStamp{}) const
+  {
+    return reduce<
+      OpT,
+      FunctorT,
+      MsgPtrT,
+      MsgT,
+      MsgT::template msgHandler<MsgT, OpT, FunctorT>
+      >(msg, stamp);
+  }
 
   /**
    * \brief Reduce over the objgroup instance on each node with target specified

--- a/src/vt/objgroup/proxy/proxy_objgroup.h
+++ b/src/vt/objgroup/proxy/proxy_objgroup.h
@@ -168,7 +168,7 @@ public:
       OpT,
       MsgPtrT,
       MsgT,
-      MsgT::template msgHandler<
+      &MsgT::template msgHandler<
         MsgT, OpT, collective::reduce::operators::ReduceCallback<MsgT>
         >
       >(msg, cb, stamp);
@@ -204,7 +204,7 @@ public:
       FunctorT,
       MsgPtrT,
       MsgT,
-      MsgT::template msgHandler<MsgT, OpT, FunctorT>
+      &MsgT::template msgHandler<MsgT, OpT, FunctorT>
       >(msg, stamp);
   }
 

--- a/src/vt/vrt/collection/reducable/reducable.h
+++ b/src/vt/vrt/collection/reducable/reducable.h
@@ -71,21 +71,49 @@ struct Reducable : BaseProxyT {
   template <
     typename OpT = collective::None,
     typename MsgT,
-    ActiveTypedFnType<MsgT> *f = MsgT::template msgHandler<
-      MsgT, OpT, collective::reduce::operators::ReduceCallback<MsgT>
-    >
+    ActiveTypedFnType<MsgT> *f
   >
   messaging::PendingSend reduce(
     MsgT *const msg, Callback<MsgT> cb, ReduceStamp stamp = ReduceStamp{}
   ) const;
+  template <
+    typename OpT = collective::None,
+    typename MsgT
+  >
+  messaging::PendingSend reduce(
+    MsgT *const msg, Callback<MsgT> cb, ReduceStamp stamp = ReduceStamp{}
+  ) const
+  {
+    return reduce<
+      OpT,
+      MsgT,
+      MsgT::template msgHandler<
+        MsgT, OpT, collective::reduce::operators::ReduceCallback<MsgT>
+        >
+      >(msg, cb, stamp);
+  }
 
   template <
     typename OpT,
     typename FunctorT,
     typename MsgT,
-    ActiveTypedFnType<MsgT> *f = MsgT::template msgHandler<MsgT, OpT, FunctorT>
+    ActiveTypedFnType<MsgT> *f
   >
   messaging::PendingSend reduce(MsgT *const msg, ReduceStamp stamp = ReduceStamp{}) const;
+  template <
+    typename OpT,
+    typename FunctorT,
+    typename MsgT
+  >
+  messaging::PendingSend reduce(MsgT *const msg, ReduceStamp stamp = ReduceStamp{}) const
+  {
+    return reduce<
+      OpT,
+      FunctorT,
+      MsgT,
+      MsgT::template msgHandler<MsgT, OpT, FunctorT>
+      >(msg, stamp);
+  }
 
   template <typename MsgT, ActiveTypedFnType<MsgT> *f>
   messaging::PendingSend reduce(

--- a/src/vt/vrt/collection/reducable/reducable.h
+++ b/src/vt/vrt/collection/reducable/reducable.h
@@ -87,7 +87,7 @@ struct Reducable : BaseProxyT {
     return reduce<
       OpT,
       MsgT,
-      MsgT::template msgHandler<
+      &MsgT::template msgHandler<
         MsgT, OpT, collective::reduce::operators::ReduceCallback<MsgT>
         >
       >(msg, cb, stamp);
@@ -111,7 +111,7 @@ struct Reducable : BaseProxyT {
       OpT,
       FunctorT,
       MsgT,
-      MsgT::template msgHandler<MsgT, OpT, FunctorT>
+      &MsgT::template msgHandler<MsgT, OpT, FunctorT>
       >(msg, stamp);
   }
 

--- a/tests/unit/active/test_active_send_large.cc
+++ b/tests/unit/active/test_active_send_large.cc
@@ -135,8 +135,9 @@ TYPED_TEST_P(TestActiveSendLarge, test_large_bytes_msg) {
   auto e = pipe::LifetimeEnum::Once;
   auto cb = theCB()->makeFunc<RecvMsg>(e, [&counter](RecvMsg*){ counter++; });
 
+  NodeType next_node = (this_node + 1) % num_nodes;
+
   vt::runInEpochCollective([&]{
-    NodeType next_node = (this_node + 1) % num_nodes;
     auto msg = makeMessage<LargeMsgType>();
     fillMsg(msg);
     msg->cb_ = cb;

--- a/tests/unit/test_parallel_harness.h
+++ b/tests/unit/test_parallel_harness.h
@@ -77,7 +77,7 @@ struct MPISingletonMultiTest {
   }
 
 public:
-  const MPI_Comm getComm() { return comm_; }
+  MPI_Comm getComm() { return comm_; }
 
 private:
   MPI_Comm comm_;

